src/main/java/uk/gov/companieshouse/api/accounts/validation/CreditorsAfterOneYearValidator.java

```diff
@@ -70,26 +97,31 @@ public Errors validateCreditorsAfterOneYear(@Valid CreditorsAfterOneYear credito
         return errors;
     }
 
-    private void validateCurrentPeriod(CurrentPeriod currentPeriodNote, BalanceSheet currentPeriodBalanceSheet, Errors errors) {
+    private void validateCurrentPeriod(CurrentPeriod currentPeriodNote,
+            BalanceSheet currentPeriodBalanceSheet, Errors errors) {
 
         boolean hasCurrentPeriodBalanceSheet = currentPeriodBalanceSheet != null;
-        boolean hasCurrentPeriodBalanceSheetNoteValue = !isCurrentPeriodBalanceSheetDataNull(currentPeriodBalanceSheet);
+        boolean hasCurrentPeriodBalanceSheetNoteValue =
+                ! isCurrentPeriodBalanceSheetDataNull(currentPeriodBalanceSheet);
         boolean hasCurrentPeriodNoteData = currentPeriodNote != null;
 
-        if (!hasCurrentPeriodBalanceSheetNoteValue && hasCurrentPeriodNoteData) {
+        if (! hasCurrentPeriodBalanceSheetNoteValue && hasCurrentPeriodNoteData) {
 
-            if (validateNoUnexpectedDataPresent(hasCurrentPeriodBalanceSheet, CREDITORS_AFTER_CURRENT_PERIOD_PATH, errors)) {
+            if (validateNoUnexpectedDataPresent(hasCurrentPeriodBalanceSheet,
+                    CREDITORS_AFTER_CURRENT_PERIOD_PATH, errors)) {
                 validateCurrentPeriodFields(currentPeriodNote, errors);
             }
 
-        } else if (validateCurrentPeriodExists(hasCurrentPeriodBalanceSheetNoteValue, hasCurrentPeriodNoteData, errors)) {
+        } else if (validateCurrentPeriodExists(hasCurrentPeriodBalanceSheetNoteValue,
+                hasCurrentPeriodNoteData, errors)) {
 
             if (hasCurrentPeriodNoteData) {
                 validateCurrentPeriodFields(currentPeriodNote, errors);
             }
 
             if (hasCurrentPeriodNoteData && hasCurrentPeriodBalanceSheetNoteValue) {
-                crossValidateCurrentPeriodFields(currentPeriodNote, currentPeriodBalanceSheet, errors);
+                crossValidateCurrentPeriodFields(currentPeriodNote, currentPeriodBalanceSheet,
```

Comment:
This if statement is always true.

Reply:
It will be false if there is no balance sheet submitted, or no corresponding note value (eg creditors) is entered on the balance sheet and you try and submit the note (will only ever be false when submitting via api)

Reply:
The if statement on line 108 has `if (! hasCurrentPeriodBalanceSheetNoteValue &&` which ensures that `hasCurrentPeriodBalanceSheetNoteValue` is true by the time you reach line 122.